content/id/docs/reference/access-authn-authz/rbac.md

```diff
@@ -1185,10 +1185,11 @@ memuat [kebijakan ABAC lama](/docs/reference/access-authn-authz/abac/#policy-fil
 --authorization-mode=...,RBAC,ABAC --authorization-policy-file=mypolicy.json
 ```
 
-To explain that first command line option in detail: if earlier authorizers, such as Node,
-deny a request, then the RBAC authorizer attempts to authorize the API request. If RBAC
-also denies that API request, the ABAC authorizer is then run. This means that any request
-allowed by *either* the RBAC or ABAC policies is allowed.
```

Comment:
Can I know why are we changing the english content to other language? 
@altmas5